docs/swagger.yaml

```diff
@@ -12,7 +12,7 @@ servers:
   - url: 'https://api.vipps.no/order-management'
     description: Vipps Order Management API
 paths:
-  '/v1/orders/{transactionId}':
+  '/order-management/v1/orders/{transactionId}':
```

Comment:
we cant add order-management in front of the path, we have order management in the base path of the server
<img width="610" alt="image" src="https://user-images.githubusercontent.com/47554984/196902704-420efa42-895e-4c2a-8287-32793fbe1d2c.png">